### ISSUE TITLE
Add toggle to enable or disable a monitor and add bulk actions

### ISSUE THREAD

**Issue by onedr0p** [OneUptime/oneuptime#560]
**Is your feature request related to a problem? Please describe.**

It shouldn't take 5 clicks to enable or disable a monitor.

**Describe the solution you'd like**

Add a enable/disable silder/toggle to this page and add bulk actions to easily change the state of monitors.

<img width="1197" alt="image" src="https://github.com/OneUptime/oneuptime/assets/213795/c2a2b244-b306-4b7c-ba76-829d4fc47459">

**Describe alternatives you've considered**

Clicking 100 times to disable 20 monitors 😄 

**Additional context**

**Comment by onedr0p** [comment]
Also, Monitor Status should report state Disabled is not enabled.

**Comment by simlarsen** [comment]
Can you please create a new GH issue for second comment.

**Comment by onedr0p** [comment]
Yup, done in https://github.com/OneUptime/oneuptime/issues/562 thanks!
